From: Stanislav Ochotnicky <sochotnicky@redhat.com>
Date: Wed, 12 Nov 2014 20:48:28 +0100
Subject: Fix qt4 pinentry window created in the background

This is probably just a workaround. Proper fix is being investigated.
See:
https://bugzilla.redhat.com/show_bug.cgi?id=589532
http://stackoverflow.com/questions/2788518/calling-activatewindow-on-qdia

dkg added these additional references:
 - https://bugs.gnupg.org/gnupg/issue1981
 - https://bugs.debian.org/581617
---
 qt4/pinentrydialog.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/qt4/pinentrydialog.cpp b/qt4/pinentrydialog.cpp
index 456f022..89735c2 100644
--- a/qt4/pinentrydialog.cpp
+++ b/qt4/pinentrydialog.cpp
@@ -76,7 +76,6 @@ void raiseWindow( QWidget* w )
     /* Maybe Qt will become agressive enough one day that
      * this is enough on windows too*/
     w->raise();
-    w->activateWindow();
 #ifdef Q_WS_WIN
     /* In the meantime we do our own attention grabbing */
     if (!SetForegroundWindow (w->winId()) &&
